src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
```
Health is 100 by default.
But we can pass to the constructor another value if needed